.mailmap

```diff
@@ -0,0 +1,32 @@
+Nils Tonnätt <nils.tonnaett@posteo.de>
+Nils Tonnätt <nils.tonnaett@posteo.de> studio <akkijyrkkae@campus.tu-berlin.de>
+
+Marcin Pączkowski <dyfeer@gmail.com>
+
+Maximilian Wagenbach <maximilian.wagenbach@native-instruments.de>
+
+Julius Smith <jos@ccrma.stanford.edu>
+Julius Smith <jos@ccrma.stanford.edu> Julius Smith <julius.smith@gmail.com>
+
+Juan-Pablo Caceres <caceres@adobe.com> <jcacerec@gmail.com>
+Juan-Pablo Caceres <caceres@adobe.com> <jcacerec@users.noreply.github.com>
+Juan-Pablo Caceres <caceres@adobe.com> <jcacerec@3532cb72-8e7e-11dd-98a5-cbe9a756ba3b>
+Juan-Pablo Caceres <caceres@adobe.com> jcaceres <jcaceres@3532cb72-8e7e-11dd-98a5-cbe9a756ba3b>
+Juan-Pablo Caceres <caceres@adobe.com> jua20133 <jua20133@adobe.com>
+Juan-Pablo Caceres <caceres@adobe.com> jcaceres <jcaceres@ccrma.stanford.edu>
+
+Aaron Wyatt <github@psi-borg.org
```

Comment:
should there be a > at the end of that line?

Reply:
Maybe…